src/Options.php

```diff
@@ -729,6 +771,8 @@ private function configureOptions(OptionsResolver $resolver): void
             'capture_silenced_errors' => false,
             'max_request_body_size' => 'medium',
             'class_serializers' => [],
+            'http_connect_timeout' => HttpClientFactory::DEFAULT_HTTP_CONNECT_TIMEOUT,
+            'http_timeout' => HttpClientFactory::DEFAULT_HTTP_TIMEOUT,
```

Comment:
Also, I'm not sure about the option names. We should check if there's an unified name already used in other Sentry SDKs.

Reply:
I had a look at the python SDK and the only references I could find where `timeout`, here we're being a bit more explicit, happy to change it if you like.